lib/OrgChart.tsx

```diff
@@ -441,25 +442,6 @@ export default class OrgChart<T> extends React.Component<
     return { prevProps: props };
   }
 
-  componentWillMount() {
```

Comment:
Not sure if I'm breaking some backward compatibility case by removing this, thus, for my react typescript environment, I had to totally remove componentWillMount, and componentWillReceiveProps , in order to avoid errors.